internal/server/server.go

```diff
@@ -172,3 +174,31 @@ func (s ServerForClient) GetGameState() gamestate.ClientGameState {
 func (s ServerForClient) GetGameConfig() config.ClientConfig {
 	return s.server.gameConfig.GetClientConfig()
 }
+
+func randomAssign(input []shared.ClientID) (shared.ClientID, shared.ClientID, shared.ClientID) {
```

Comment:
nitpick: I'd rename this to `randomAssignIIGORoles`, `randomAssign` is a very generic name.

Reply:
Done